src/App/Hovedside/Hovedside.less

```diff
@@ -2,10 +2,38 @@
   display: flex;
   flex-direction: column;
   background-color: #EFEFEF;
+  height: 45rem;
+
+  .boks {
```

Comment:
Smart å generalisere stylingen for boksene. Passer kanskje bedre å generalisere i en egen komponent. Alle boksene vi har kan da wrappes i denne komponenten så får den riktig styling med hvit bakgrunn og samme padding.
Sjekk komponenten [her](https://github.com/navikt/tiltaksgjennomforing/blob/master/src/komponenter/Innholdsboks/Innholdsboks.tsx) og [stylingen](https://github.com/navikt/tiltaksgjennomforing/blob/master/src/komponenter/Innholdsboks/Innholdsboks.less)